app/entry.client.tsx

```diff
@@ -5,6 +5,9 @@ import { hydrateRoot } from 'react-dom/client'
 if (ENV.MODE === 'development') {
 	import('~/utils/devtools').then(({ init }) => init())
 }
+if (ENV.MODE === 'production') { 
+	import('~/utils/monitoring.client').then(({ init }) => init())
```

Comment:
I'm not sure if this is the best way to do this but I just copied what we do for devtools

Reply:
@kentcdodds thoughts on this strategy?